l2geth/rollup/sync_service.go

```diff
@@ -266,16 +266,12 @@ func (s *SyncService) Start() error {
 
 	if s.verifier {
 		go s.VerifierLoop()
-	} else {
+	} else if !s.IsSequencerMode() {
 		go func() {
 			if err := s.syncTransactionsToTip(); err != nil {
 				log.Crit("Sequencer cannot sync transactions to tip", "err", err)
 			}
-			if err := s.syncQueueToTip(); err != nil {
-				log.Crit("Sequencer cannot sync queue to tip", "err", err)
-			}
 			s.setSyncStatus(false)
```

Comment:
For a sequencer node, do you think `s.setSyncStatus(false)` is required? If sync status is always true, sequencer node can't receive tx from rpc.